trino/client.py

```diff
@@ -555,9 +558,20 @@ def cancel(self):
         self._request.raise_response_error(response)
 
     def is_finished(self):
+        import warnings
+        warnings.warn("is_finished is deprecated, use finished instead", DeprecationWarning)
+        return self.finished
```

Comment:
Why we change `is_finished` -> `finished` in this PR?
Why are you adding `cancelled` property? Is this some API we're filling?

Reply:
I believe `cancelled` should be public, not just internal use (like `is_finished()`).
And I also change from `is_finished` to `finished` to align with new `cancelled` properties. You could see previous discussion with @joshthoward